tls/handshake.py

```diff
@@ -256,7 +256,6 @@ def FINAL(self):
         # did not wait for us to shutdown after answering our data query.
         # self.socket.shutdown(1)
         tf_cfg.dbg(2, "Closing client socket...")
-        self.socket.close()
         tf_cfg.dbg(2, "Ending TLS client automaton.")
```

Comment:
If FINAL state hasn't been reached, we got file-descriptors leak, cause Automaton will not stop properly

Reply:
I got it:
If valid ciphersuite like TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 not presented in first 122 - tempesta returns an error:
`Warning: [::ffff:127.0.0.1] None of the common ciphersuites is usable (e.g. no suitable certificate), like TLS-DHE-RSA-WITH-AES-256-GCM-SHA384`

In master i found dirty trick to allow `list(range(2000))` work properly:

https://github.com/tempesta-tech/tempesta-test/blob/a58db54c45109cf150881cbec69a21e09a13fddf/tls/handshake.py#L285

So i think we should fix it by tempesta side or i can make workaround to avoid errors in test